### PR TITLE
Resolve Black Screen on Intel GPU Regression

### LINKED ISSUE #6278: Black screen during intro of radiant historia [INTEL REGRESSION FROM #6247]
On recent canary build the game started with black screen during intro of the game but then everything loaded normally during actual gameplay. resetting the game after that also let the intro loads normally but after closing citra the game started with the black screen intro again & repeat. the game loads normally on the latest nightly build

### PR DESCRIPTION
**Pull request by SomeDudeOnDiscord**
Attempt to resolve #6278, which is an issue unique to Intel GPUs

- - -
This change is [Reviewable](https://reviewable.io/reviews/citra-emu/citra/6306)

### CODE REVIEW

**Comment by SomeDudeOnDiscord** [comment]
After testing these changes, I found that the issue remains unresolved. I am open to suggestions.

**Comment by SomeDudeOnDiscord** [comment]
With the latest commit, I am no longer able to reproduce the black screen issue. However, a new one has popped up. The custom_second_screen_opacity setting now applies to both screens instead of just the second screen. I am still looking into this.

**Comment by SomeDudeOnDiscord** [comment]
I am no longer able to reproduce the other issue with the latest commit. Thank you @GPUCode for your help!

File: src/video_core/renderer_opengl/renderer_opengl.cpp [hunk 1]
```diff
@@ -1026,6 +1027,14 @@ void RendererOpenGL::ApplySecondLayerOpacity() {
     }
 }
 
+void RendererOpenGL::ResetSecondLayerOpacity() {
+    state.blend.src_rgb_func = GL_ONE;
```
Comment (GPUCode) [inline src/video_core/renderer_opengl/renderer_opengl.cpp:1031]: Wrap this in the same ifs as ApplySecondLayerOpacity so it doesn't get executed when the opacity option is not tampered with

**Pull request merged**